packages/misc/src/lib/misc.ts

```diff
@@ -13,29 +14,24 @@ import {
 import {
   Chain,
   AuthSig,
-  KV,
   NodeClientErrorV0,
   NodeClientErrorV1,
-  NodeErrorV1,
   NodeErrorV3,
-  ClaimRequest,
-  ClaimKeyResponse,
   ClaimResult,
-  ClaimProcessor,
   MintCallback,
   RelayClaimProcessor,
-  SuccessNodePromises,
-  RejectedNodePromises,
-  RetryTolerance,
 } from '@lit-protocol/types';
 import { JsonRpcProvider } from '@ethersproject/providers';
 import { Contract } from '@ethersproject/contracts';
 import { LogLevel, LogManager } from '@lit-protocol/logger';
-import { version } from '@lit-protocol/constants';
 import Ajv, { JSONSchemaType } from 'ajv';
+//@ts-ignore no tyoe df
+import fetchRetry from 'fetch-retry';
```

Comment:
Hmm I'm worried how browser (react, vite, parcel, etc.) would react to this dependency, I'll give it a spin next week

Reply:
its bundle is 'umd' format. I think it's okay but I'll publish a testing build with it for us to test with more before committing to it.